src/OrchardCore.Modules/OrchardCore.Queries/Controllers/ApiController.cs

```diff
@@ -43,12 +43,30 @@ IQueryManager queryManager
                 return NotFound();
             }
 
-            var queryParameters = parameters != null ?
+            string parameters;
+            if (Request.Method == "GET")
+            {
+                parameters = Request.Query["parameters"].ToString();
+            }
+            else if (Request.Method == "POST")
```

Comment:
This might be a breaking change. Before even in a POST the `parameters` query string would have been read, now it's not.

Instead could you check if the qyery string is set, and if not use the body when in a POST.

Reply:
I think I fixed it. If the parameters are empty as query string in Post type, I get the request body value.

This way I can pass the long parameters I need.

Example: 
`{
    "EmployeeIds": "4s726tsb5jpdhrny3v0759jjw0,4znzn2eecdazvrc2kh2vhm4yn9,4ggs0tgqrat127jgsrps1rr75g,4dts0wxjsp5p1saeprkk67redt,4dts0wxjsp5p1saeprkk67redt,48bweg5d4chwn08yzc1brwys5v"
}`